### ISSUE TITLE
Incremental topic 4 - 4.C and 4.F

### ISSUE THREAD

**Issue by Tonylayton1** [Section508Coordinators/DHSA-TT-200-A#418]
Only one end of the keyboard trap was counting towards 4.F. Add r2e04 to the list of failures to 4.F when the keyboard trap occurs.

**Comment by mikebarklage** [comment]
Done.

**Comment by AnnMarieDavis** [comment]
Unless we get different instructions from the U.S. Access Board, we will fail both forward and backward focus order. We may need to revisit this at a later date.
